core/jazz_login/package.json

```diff
@@ -1,23 +0,0 @@
-{
-    "name": "login",
```

Comment:
You need to remove all other micro services from this PR which includes the login service. But here you are deleting the file and not removing it from PR. Please take Mani's help here since we have been talking about this for sometime now.